lib/embiggen/uri.rb

```diff
@@ -43,8 +43,17 @@ def follow(request_options = {})
       location = http_client.follow(timeout)
       fail BadShortenedURI.new(
         "following #{uri} did not redirect", uri) unless location
+      check_location(location)
```

Comment:
I wonder if this is the same as the line above: rather than having `unless location` *and* `check_location(location)`, could they be combined into one validity check?